### PR TITLE
added header ,footer and social media icons

### PR DESCRIPTION
**Pull request by deepakverma05**

### CODE REVIEW

File: src/components/footer.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import '../styles/_footer.scss'
+
+const Footer = () => (
+  <footer className={`footer`}>
+      <div className={`footer__content-wrapper`}>
+          <div className={`footer__copyright`}>
+          © {new Date().getFullYear()}, {` `}
```
Comment (spacebaboon) [inline src/components/footer.js:8]: we can just hard code the year

**Comment by akshaymayekar** [comment]
LGTM

File: src/styles/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
+$footer-background:#373d45;
```
Comment (spacebaboon) [inline src/styles/_footer.scss:1]: do we need a separate scss file for each component? there are some duplications of styles already between the header and footer.

**Review by spacebaboon — CHANGES_REQUESTED** [review]
Looks good. Just a couple of things to fix.

**Pull request merged**